CONTAINERS.md

```diff
@@ -32,20 +32,38 @@ It also runs database migrations.
 
 ## Hyrax Image
 
-We also provide a base image which can be reused for Hyrax applications: `hyrax`.
+We also provide a base image which can be reused for your Hyrax applications: `hyrax`.
 
 ```sh
-echo "FROM hyrax" > Dockerfile
+echo "FROM samveralabs/hyrax" > Dockerfile
 ```
 
+_This is for applications that mount Hyrax and is separate from the docker containers for Hyrax engine development._
```

Comment:
one of the things i hope this documentation will eventually capture is that these containers are separate, in that we have to take action to maintain them separately; but also that they are closely related. the two images use the same base, and (hopefully) the `hyrax-engine-dev` is the same as the `hyrax` image in every substantial way.